src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -67,17 +70,20 @@ class MovingWindow:
     time_end = time_start + timedelta(minutes=5)
 
     # ... wait for 5 minutes until the buffer is filled
-    await asyncio.sleep(5)
-
-    # return an numpy array from the window
-    a = window[time_start:time_end]
-    # and use it to for example calculate the mean
-    mean = a.mean()
-    '''
+    await asyncio.sleep(0.1)
```

Comment:
Maybe 0.1 is too little?

Reply:
I mean, probably, but it does cause actual wait times when running it. If we want to put realistic values here then I would try to make it use fake-event loops